### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by munakaya** [tezos/tezos#153]
make generates an error.
It there somthing that I can check ?

make
....

      ocamlc lib_base/tezos_base__TzPervasives.{cmi,cmti}
    ocamlopt lib_stdlib_lwt/tezos_stdlib_lwt__Lwt_exit.{cmx,o}
      ocamlc lib_micheline/tezos_micheline__Micheline.{cmi,cmti}
      ocamlc lib_protocol_compiler/main_embedded_packer.{cmi,cmti}
      ocamlc lib_node_p2p_base/tezos_node_p2p_base__P2p_types.{cmi,cmti}
    ocamlopt lib_stdlib_lwt/tezos_stdlib_lwt__Lwt_utils.{cmx,o} (exit 2)
(cd _build/default && /home/user/.opam/tezos/bin/ocamlopt.opt -w @a-4-29-40-41-42-44-45-48-58-59-60-40 -strict-sequence -strict-formats -short-paths -keep-locs -w -9-30 -open Tezos_stdlib -open Tezos_data_encoding -open Tezos_error_monad -safe-string -g -I /home/user/.opam/tezos/lib/bytes -I /home/user/.opam/tezos/lib/cstruct -I /home/user/.opam/tezos/lib/ipaddr -I /home/user/.opam/tezos/lib/ipaddr/unix -I /home/user/.opam/tezos/lib/lwt -I /home/user/.opam/tezos/lib/ocaml -I /home/user/.opam/tezos/lib/ocaml/threads -I /home/user/.opam/tezos/lib/ocplib-endian -I /home/user/.opam/tezos/lib/ocplib-json-typed -I /home/user/.opam/tezos/lib/re -I /home/user/.opam/tezos/lib/result -I /home/user/.opam/tezos/lib/sexplib -I /home/user/.opam/tezos/lib/sexplib/0 -I /home/user/.opam/tezos/lib/stringext -I /home/user/.opam/tezos/lib/uri -I lib_data_encoding -I lib_error_monad -I lib_stdlib -no-alias-deps -I lib_stdlib_lwt -open Tezos_stdlib_lwt -o lib_stdlib_lwt/tezos_stdlib_lwt__Lwt_utils.cmx -c -impl lib_stdlib_lwt/lwt_utils.ml)
File "lib_stdlib_lwt/lwt_utils.ml", line 90, characters 8-29:
Warning 3: deprecated: Lwt.waiter_of_wakener
 This function should be avoided, because it makes subtyping of resolvers
 unsound. See
  https://github.com/ocsigen/lwt/issues/458
File "lib_stdlib_lwt/lwt_utils.ml", line 119, characters 8-29:
Warning 3: deprecated: Lwt.waiter_of_wakener
 This function should be avoided, because it makes subtyping of resolvers
 unsound. See
  https://github.com/ocsigen/lwt/issues/458
File "lib_stdlib_lwt/lwt_utils.ml", line 1:
Error: Some fatal warnings were triggered (2 occurrences)
Makefile:5: recipe for target 'all' failed
make: *** [all] Error 1

**Comment by tzs1** [comment]
I get this error:

```
ocamlc src/node/net/RPC_server.{cmo,cmt} (exit 2)
(cd _build/default && /home/tzs/.opam/tezos/bin/ocamlc.opt -w @a-4-29-40-41-42-44-45-48-58-59-60-40 -strict-sequence -strict-formats -short-paths -keep-locs -w -9+27-30-32-40@8 -safe-string -open Error_monad -open Hash -open Utils -open Tezos_data -g -bin-annot -I /home/tzs/.opam/tezos/lib/astring -I /home/tzs/.opam/tezos/lib/base -I /home/tzs/.opam/tezos/lib/base/caml -I /home/tzs/.opam/tezos/lib/base/shadow_stdlib -I /home/tzs/.opam/tezos/lib/base64 -I /home/tzs/.opam/tezos/lib/bytes -I /home/tzs/.opam/tezos/lib/calendar -I /home/tzs/.opam/tezos/lib/cohttp -I /home/tzs/.opam/tezos/lib/cohttp-lwt -I /home/tzs/.opam/tezos/lib/cohttp-lwt-unix -I /home/tzs/.opam/tezos/lib/conduit -I /home/tzs/.opam/tezos/lib/conduit-lwt -I /home/tzs/.opam/tezos/lib/conduit-lwt-unix -I /home/tzs/.opam/tezos/lib/cstruct -I /home/tzs/.opam/tezos/lib/ctypes -I /home/tzs/.opam/tezos/lib/ezjsonm -I /home/tzs/.opam/tezos/lib/fieldslib -I /home/tzs/.opam/tezos/lib/fmt -I /home/tzs/.opam/tezos/lib/hex -I /home/tzs/.opam/tezos/lib/integers -I /home/tzs/.opam/tezos/lib/ipaddr -I /home/tzs/.opam/tezos/lib/ipaddr/unix -I /home/tzs/.opam/tezos/lib/jsonm -I /home/tzs/.opam/tezos/lib/logs -I /home/tzs/.opam/tezos/lib/lwt -I /home/tzs/.opam/tezos/lib/lwt_ssl -I /home/tzs/.opam/tezos/lib/magic-mime -I /home/tzs/.opam/tezos/lib/mtime -I /home/tzs/.opam/tezos/lib/mtime/os -I /home/tzs/.opam/tezos/lib/nocrypto -I /home/tzs/.opam/tezos/lib/ocaml -I /home/tzs/.opam/tezos/lib/ocaml/threads -I /home/tzs/.opam/tezos/lib/ocplib-endian -I /home/tzs/.opam/tezos/lib/ocplib-json-typed -I /home/tzs/.opam/tezos/lib/ocplib-resto -I /home/tzs/.opam/tezos/lib/ocplib-resto/directory -I /home/tzs/.opam/tezos/lib/re -I /home/tzs/.opam/tezos/lib/result -I /home/tzs/.opam/tezos/lib/sexplib -I /home/tzs/.opam/tezos/lib/sexplib/0 -I /home/tzs/.opam/tezos/lib/sodium -I /home/tzs/.opam/tezos/lib/ssl -I /home/tzs/.opam/tezos/lib/stringext -I /home/tzs/.opam/tezos/lib/uchar -I /home/tzs/.opam/tezos/lib/uri -I /home/tzs/.opam/tezos/lib/uri/services -I /home/tzs/.opam/tezos/lib/uutf -I /home/tzs/.opam/tezos/lib/zarith -I src/minutils -I src/utils -no-alias-deps -I src/node/net -o src/node/net/RPC_server.cmo -c -impl src/node/net/RPC_server.ml)
File "src/node/net/RPC_server.ml", line 95, characters 21-42:
Error: Unbound module Cohttp_lwt_body
```

**Comment by MiloDavis** [comment]
This is fixed as of 58ad60f38a4c4f9c084f03d04e57eef497648abd.
